packages/atclient/include/atclient/atsign.h

```diff
@@ -1,14 +1,32 @@
 #ifndef ATCLIENT_ATSIGN_H
 #define ATCLIENT_ATSIGN_H
 
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include <ctype.h>
+
+// Structure to represent the AtSign class
+typedef struct atclient_atsign {
+    char* atsign;
+    char* without_prefix_str;
+} atclient_atsign;
+
+// Function to initialize an AtSign object
+int atclient_atsign_init(atclient_atsign *atsign, char *atsign_str);
+
```

Comment:
Since we don't modify `char *atsign_str`, ideally this parameter would be `const char *atsign_str` to indicate we only plan on reading this argument.

Reply:
Solved https://github.com/atsign-foundation/at_c/pull/111/commits/584d8d8f52f57716346dba973d9149b8292a378f